docs/source/sctool/task.rst

```diff
@@ -17,21 +17,23 @@ The task command set allows you to schedule, start, stop and modify tasks.
 
    * - Subcommand
      - Usage
-   * - `task delete`_
+   * - :ref`task delete<task-delete>`
      - Delete a task.
-   * - `task history`_
+   * - :ref`task history<task-history>`
      - Show run history of a task.
-   * - `task list`_
+   * - :ref`task delete<task-delete>`
      - Show available tasks and their last run status.
-   * - `task progress`_
+   * - :ref`task progress<task-progress>`
      - Show the task progress.
-   * - `task start`_
+   * - :ref`task start<task-start>`
      - Start executing a task.
-   * - `task stop`_
+   * - :ref`task stop<task-stop>`
      - Stop executing a task.
-   * - `task update`_
+   * - :ref`task update<task-update>`
      - Modify a task.
 
+.. _task-delete:
+
 task delete
```

Comment:
@mmatczuk I couldn't find an equivalent command for ``task delete``. Should we delete it from the docs (rst)?